hugegraph-server/hugegraph-dist/src/assembly/travis/test-check-port.sh

```
@@ -0,0 +1,407 @@
+    # Use pd's util.sh instead of server's for this specific test
+    PD_UTIL_SH="hugegraph-pd/hg-pd-dist/src/assembly/static/bin/util.sh"
+    if [[ ! -f "$PD_UTIL_SH" ]]; then
+        # If running from a different directory, try relative to STATIC_DIR
+        PD_UTIL_SH="$(dirname "$STATIC_DIR")/../../../../hugegraph-pd/hg-pd-dist/src/assembly/static/bin/util.sh"
+    fi
+    if [[ ! -f "$PD_UTIL_SH" ]]; then
+        echo "Could not find pd util.sh to test download(), skipping."
+        exit 0
+    fi
+    
+    source "$PD_UTIL_SH"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)